data/World/Water Temple.json

```diff
@@ -12,7 +12,13 @@
                 # Ensure that the water level can be raised if it were to be lowered.
         },
         "locations": {
-            "Fairy Pot": "Raise_Water_Level and has_bottle and can_use(Longshot)"
+            "Fairy Pot": "Raise_Water_Level and has_bottle and can_use(Longshot)",
+            "Water Temple Main Room L2 Pot 1" : "at('Water Temple Middle Water Level', 'True') or
+                ((can_use(Iron_Boots) or (Progressive_Scale, 2)) and (can_use(Bow) or can_use(Hookshot)))", 
+            "Water Temple Main Room L2 Pot 2" : "at('Water Temple Middle Water Level', 'True') or
+                ((can_use(Iron_Boots) or (Progressive_Scale, 2)) and (can_use(Bow) or can_use(Hookshot)))",
```

Comment:
Middle Water Level access includes a method to get into the central pillar in order to bring the water up to the middle level. Since you can just jump to these pots from the entrance once the water is lowered, this isn't needed. You instead want to be at Water Temple Dive with Zelda's Lullaby.

Child can Boomerang these pots and not need to dive down.
Child can also use the Slingshot and gold scale.
(As both ages these pots can be blown up with a bombchu trick. Not worth implementing at this time, of course.)

Reply:
While I understand the sentiment, I don't agree with these particular checks requiring tunic. For these checks with iron boots, the timer doesn't even start until after you reach the platform containing the pots.

Reply:
Back in the day the entire temples always required the tunic at all times. During 3.0 dev, the tunic requirements were changed to about what FTR is now. Before release that was walked back and fewer tunic requirements became a trick. Without fewer tunic requirements on, again the whole of both temples were tunic locked. This type of logic doesn't make much sense anymore now that ER is a thing and you might randomly appear in these Temples with access to checks without going into any timed areas. I felt the best way to be consistent with the original intent of FTR, which was to require the tunic for the entire temple, was to require it any at time when going into a heated room or going underwater might be required.

Reply:
Requested Change: You put the closing quotation on the can_play ZL in the wrong place on both of these pots.

Reply:
On second thought, you don't normally put the condition in quotes. I don't know whether that will actually be a problem or not.

Reply:
This thread should not have been resolved as the issue with zora tunic/ftr on Main Room L2 Pots has not been resolved.